frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -211,219 +230,290 @@ describe("QueryBuilder", () => {
     });
   });
 
-  describe("editing models", () => {
-    describe("editing queries", () => {
+  describe("renders the row count regardless of visualization type", () => {
```

Comment:
This is only moved from below. Because it's about testing rendering so I move it closer to another rendering test.

Reply:
What do you think about putting these `rendering` tests inside its own describe block?

Reply:
That's a great idea. That's a nice way to make sure we run different kinds of tests
1. Rendering
2. onbeforeunload
3. downloading

Reply:
Fixed in https://github.com/metabase/metabase/pull/32251/commits/ac497a2e5e19b8711d264e113805955fbbc0eef8